---
title: "Research"
author_profile: true
classes: wide
layout: single
permalink: /research/
header:
  teaser: /assets/images/preview.png
---

<!DOCTYPE html>
<html>
    
    <link rel="stylesheet" href="/assets/css/style.css">
    
    <body>
        
        <h2 align="justify">Harnessing Microbiomes to Enhance Biological Processes</h2>
        
        <div align="justify">
            
            <p>Microorganisms are vital for several biological processes that underpin our society. They enhance agriculture through <b>plant-soil interactions</b>, drive <b>food fermentations</b>,
                and play a key role in <b>wastewater treatment</b>. Studying the ecology of these systems allows us to enhance said processes, addressing environmental challenges, and contributing 
                to a more sustainable society.</p>
            
            <img align="right" width="120px" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1b/R_logo.svg/724px-R_logo.svg.png" style="margin:10px 10px">
            
            To explore the <b>microbial ecology</b> of these systems, I became proficient in <b>bioinformatics</b>, including managing NGS sequences (amplicon sequencing, (meta-)genomics,
            (meta-)transcriptomics). I also gained expertise in multivariate statistics and network analysis using <b>R language</b>, enabling me to initiate <b>national</b> and <b>international</b> 
            collaborations.
        
        </div> 
        
        <h3 align="left">Plant Associated Microbiomes</h3>
        <div align="justify">
            
            <p>Leveraging my expertise in microbiology, ecology and bioinformatics, I am dedicated to studying and unraveling the intricacies of interactions between soil microorganisms and plants due to 
                their relevance to develop microbiome-oriented tools in sustainable agriculture.</p>
            
            <p>On one hand, we are aiming to address how domestication has changed the plant microbiome of major crops for global agriculture. This knowledge is needed to optimize the interactions between 
                plants and microbial communities occurring in the <b>rhizosphere</b>, which is where the interactions between plants and soil biota takes place fundamentally. To achieve this, we have 
                conducted a comprehensive sampling of rhizosphere and bulk soils along the biogeographic distributions of the major global crops.</p>
            
            <img align="center" src="/assets/images/CWP_map.png">
            
            <ul>
                <li>We identified a common soil <b>core microbiome</b> among wild progenitors, which created unique microhabitats within <b>ecoregions</b> that strongly influenced the soil community assembly,
                    indicating specific co-evolutionary interactions (Fern√°ndez-Alonso <i>et al.</i> <a href="https://www.researchsquare.com/article/rs-3547051/v1">2023</a>).</li>
                <li>Despite the distinct soil microbiomes and functions among wild progenitors, we identified common trends in <b>rhizosphere influence</b>, generally higher on <b>abundant</b> taxa with 
                    strong implications in <b>multifunctionality</b> (de Celis <i>et al.</i> <a href="https://doi.org/10.1111/ele.14462">2024</a>).</li>
            </ul>
        
        </div> 
        
        <h3 align="justify">Wine Yeast Communities</h3>
        
        <div align="justify">
            
            <img align="right" width="500px" src="/assets/images/Wine_diversity.png" style="margin:10px 10px">
            
            <p>Applying a <b>microbial ecology</b> perspective, we aim to unravel the distinctive role of key microbial players in <a href="https://www.semicrobiologia.org/wp-content/uploads/2023/07/5k.-Especial-Microbiologia-de-los-Alimentos.-Ecologia-e-interacciones-microbianas.-SEM_75_web.pdf">wine fermentations</a> 
                and understand their interactions within the community and with the environment. Leveraging advanced <b>multi-omics</b> approaches, we explore the rich diversity of wine yeasts 
                (de Celis <i>et al.</i> <a href="https://academic.oup.com/lambio/article/68/6/580/6699300">2019</a>, <a href="https://academic.oup.com/femsyr/article/22/1/foac034/6648099">2022</a>;
                Belda <i>et al.</i> <a href="https://linkinghub.elsevier.com/retrieve/pii/B9780081005965228758">2021</a>; 
                Ruiz <i>et al.</i> <a href="https://ami-journals.onlinelibrary.wiley.com/doi/full/10.1111/1462-2920.15540">2021</a>, 
                shedding light on the importance of the yeast-yeast interactions in shaping wine flavor (Ruiz <i>et al.</i> <a href="https://www.sciencedirect.com/science/article/pii/S0963996920306888">2020</a>).</p>
            
            <p>Moving forward, we examine the contribution of individual wine yeast strains the overall wine composition, a crucial step in managing microbial processes with biotechnological applications 
                (Ruiz <i>et al.</i> <a href="https://www.embopress.org/doi/full/10.15252/msb.202311613">2023</a>). Deciphering the <b>molecular determinants</b> governing individual contribution of yeasts to wine 
                composition not only enhances our understanding but also guides targeted improvements of wine quality (de Celis <i>et al.</i> <a href="https://doi.org/10.1186/s40168-024-01930-w">2024</a>).</p>
            
            <p>Once the complexity of wine communities, the relevance of yeast-yeast interactions and their individual contributions to wine composition are sufficiently understood, it is an exciting idea to 
                explore the use of a synthetic <i>Saccharomyces</i> chassis for the construction of a <b>synthetic yeast</b> with genes covering the ecological functions observed in multi-species spontaneous 
                wine fermentations (Belda <i>et al.</i> <a href="https://www.nature.com/articles/s41467-021-21877-y">2021</a>).</p>
        
        </div> 
        
        <h3 align="justify">Waste Water Treatment Plants</h3>
        
        <div align="justify">
            
            <img align="left" width="350px" src="/assets/images/WWTP_network.png" style="margin:10px 10px">
            
            <p>The operation of Wastewater Treatment Plants (WWTP) depends on the activity of the <b>microbial community</b> for the degradation of organic matter and nutrients present in the wastewater. Our work 
                deepens the knowledge of microbial <b>dynamics</b> within bioreactors, studying the patterns of microbial <b>assembly</b> and <b>succession</b> (de Celis <i>et al.</i> 
                <a href="https://www.nature.com/articles/s41598-020-61092-1">2020</a>, <a href="https://www.nature.com/articles/s41522-022-00291-2">2022</a>), and advancing the development of methodologies for 
                monitoring the <b>stability</b> of the communities and the correct WWTP operation. We further identify microbial <b>associations</b> and their <b>ecological implications</b> occurring during 
                granule development in specific bioreactor settings, especially those involving <b>inter-kingdom interactions</b> (de Celis <i>et al</i>. <a href="https://www.biorxiv.org/content/10.1101/2024.01.16.575656v1.full.pdf">2024</a>).</p>
            
            <p>Combining <b>classical microbiology</b> and <b>NGS techniques</b> we identify the most promising tools to reduce <b>biofouling</b> issues in WWTPs, such as the use of <i>quorum quenching</i> 
                enzymes that inhibit biofilm formation (de Celis <i>et al.</i> <a href="https://www.sciencedirect.com/science/article/pii/S0048969721044740">2021</a>) or metallic nanoparticles with limited 
                effectiveness (de Celis <i>et al.</i> <a href="https://www.sciencedirect.com/science/article/pii/S0048969722010075">2022</a>).</p>
        
        </div> 
    
    </body>
</html>