---
layout: single
classes: wide
author_profile: true
header:
  image: /assets/images/biofilm2.png

---

<!DOCTYPE html>
<html>

<head>
<title>Miguel de Celis</title>

<h1 align="left">Miguel de Celis</h1>

</head>

<body>

<div align="justify">
 
 I'm a microbiologist, specialized in microbial ecology and bioinformatics. Postdoctoral researcher in the Sustainable Agriculture and Biogeochemistry ([SAB](https://www.ica.csic.es/index.php/en/departments/soil-plant-and-environmental-quality-department/sustainable-agriculture-and-biogeochemistry-sab)) Group of ICA-CSIC, and [MINE](http://minelab.bioucm.es/) lab graduate.
</div> 
   
<h2 align="justify">Harnessing Microbiomes to Enhance Biological Processes</h2>
<div align="justify">

Microorganisms are vital for several biological processes that underpin our society. They enhance agriculture through **plant-soil interactions**, drive **food fermentations**, and play a key role in `wastewater treatment`. Studying the ecology of these systems allows us to enhance said processes, addressing environmental challenges, and contributing to a more sustainable society.

<img align="right" height=70 src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1b/R_logo.svg/724px-R_logo.svg.png">

To explore the **microbial ecology** of these systems, I became proficient in **bioinformatics**, including managing NGS sequences (`amplicon sequencing`, `(meta-)genomics`, `(meta-)transcriptomics`). I also gained expertise in multivariate statistics and network analysis using `R language`, enabling me to initiate **national** and **international** collaborations.
</div> 
 
<h3 align="left">Plant Associated Microbiomes</h3>
<div align="justify">
 
The **MICROWILD** network aims to address how domestication has changed the plant microbiome of major crops for global agriculture. This knowledge is needed to optimize the interactions between plants and microbial communities occurring in the **rhizosphere**, which is crucial to increase global food production and its resilience to climate change.

<img align="left" height=250 src="{{site.url}}/assets/image/CWP_map.png">

-  We identified a common soil **core microbiome** among wild progenitors, which created unique microhabitats within **ecoregions** that strongly influenced the soil community assembly, indicating specific co-evolutionary interactions (Fern√°ndez-Alonso _et al._ [2023](https://www.researchsquare.com/article/rs-3547051/v1)).
-  Despite the distinct soil microbiomes and functions among wild progenitors, we identified common trends in **rhizosphere influence**, generally higher on **abundant** taxa with strong implications in **multifunctionality** (de Celis _et al._ TBA).
</div> 

<h3 align="justify">Wine Yeast Communities</h3>
<div align="justify">
 
<img align="right" height=300 src="{{site.url}}/assets/image/Wine_diversity.png">
 
Applying a **microbial ecology** perspective, we aim to unravel the distinctive role of key microbial players in [wine fermentations](https://www.semicrobiologia.org/wp-content/uploads/2023/07/5k.-Especial-Microbiologia-de-los-Alimentos.-Ecologia-e-interacciones-microbianas.-SEM_75_web.pdf) and understand their interactions within the community and with the environment. Leveraging advanced **multi-omics** approaches, we explore the rich diversity of wine yeasts (de Celis *et al.* [2019](https://academic.oup.com/lambio/article/68/6/580/6699300), [2023](https://academic.oup.com/femsyr/article/22/1/foac034/6648099); Belda *et al.* [2021](https://linkinghub.elsevier.com/retrieve/pii/B9780081005965228758); Ruiz *et al.* [2021](https://ami-journals.onlinelibrary.wiley.com/doi/full/10.1111/1462-2920.15540)), shedding light on the importance of the yeast-yeast interactions in shaping wine flavor (Ruiz *et al.* [2020](https://www.sciencedirect.com/science/article/pii/S0963996920306888)). 

Moving forward, we examine the contribution of individual wine yeast strains the overall wine composition, a crucial step in managing microbial processes with biotechnological applications (Ruiz *et al.* [2023](https://www.embopress.org/doi/full/10.15252/msb.202311613)). Deciphering the molecular determinants governing individual contribution of yeasts to wine composition not only enhances our understanding but also guides targeted improvements of wine quality (de Celis *et al.* [2023](https://www.biorxiv.org/content/10.1101/2023.12.02.569693v1)).

Once the complexity of wine communities, the relevance of yeast-yeast interactions and their individual contributions to wine composition are sufficiently understood, it is an exciting idea to explore the use of a synthetic *Saccharomyces* chassis for the construction of a synthetic yeast with genes covering the ecological functions observed in multi-species spontaneous wine fermentations (Belda *et al.* [2021](https://www.nature.com/articles/s41467-021-21877-y)).
</div> 
  
<h3 align="justify">Waste Water Treatment Plants</h3>
<div align="justify">
 
The operation of Wastewater Treatment Plants (WWTP) depends on the activity of the microbial community for the degradation of organic matter and nutrients present in the wastewater.
eepens the knowledge of microbial
dynamics in MBR bioreactors, studying the patterns of assembly and succession of
the bacterial communities in the activated sludge and relating the environmental
and operational parameters with their structure. This type of studies allows the
development of methodologies for monitoring the stability of these communities, on
which the correct operation of WWTPs depends. In addition, it also presents tools
with great potential to alleviate the biofouling of MBR membranes, such as the use
of QQ enzymes that do not negatively affect the performance of the reactor
microbiome. However, other tools studied, such as metallic NPs, have very limited
effectiveness in reducing biofilm formation, which, added to their non-specific
mechanism of action on the transcriptome of biofilm-forming bacteria, limits their
applicability in these systems and may have unpredictable consequences on the
microbial communities in the bioreactors.
</div> 


</body>
</html>